core/src/Kokkos_View.hpp

```diff
@@ -2079,9 +2081,11 @@ class View : public ViewTraits< DataType , Properties ... > {
   int use_count() const
     { return m_track.use_count(); }
 
-  KOKKOS_INLINE_FUNCTION
+  inline 
   const std::string label() const
-    { return (std::string)m_track.get_label(); }
+    { 
+       return (std::string)m_track.get_label(); 
```

Comment:
The cast should be unnecessary here.  Also, we prefer C++ - style casts to C-style casts.